.circleci/fail_fast.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Thank you for adding the bin/bash comment.

As a nit: should the comments for the contents of the script be separated from the copyright notice?

Reply:
Good idea. Done.